src/events/interactionCreate.js

```diff
@@ -28,7 +28,8 @@ module.exports = {
       });
       await command.execute(interaction);
       await logger.debug(`Executing command: ${interaction.commandName}`);
-    } catch (err) {
+    }
+ catch (err) {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 1 space.](https://app.codacy.com/gh/ZynerOrg/xyter/pullRequest?prid=9008883)
```suggestion
		catch (err) {
```